contour.yaml

```diff
@@ -0,0 +1,51 @@
+package:
```

Comment:
The file name should match the package name: `contour-1.27.yaml`

Reply:
oh yes, I forgot to change it while switching to v1.27, thanks RJ

Reply:
No problem!